code/street-banner/street-banner.js

```diff
@@ -278,6 +309,16 @@ $(document).on('knack-view-render.view_3154', function(event, page) {
     bigButton('ots-application', 'view_3154', "https://atd.knack.com/street-banners#ots-application"," " , "Over-the-Street Banner Application");
 });
 
+	//***MANAGE WALLET BUTTONS***
+$(document).on('knack-view-render.view_3673', function(event, page) {
+  // create large button on go back to Portal page
+    bigButton('manage-wallet', 'view_3673', "https://austin-tx-austin-tx.uat.cityba.se/login", "credit-card", "Manage Wallet");
+});
+$(document).on('knack-view-render.view_3675', function(event, page) {
+  // create large button on go back to Portal page
+    bigButton('manage-wallet', 'view_3675', "https://austin-tx-austin-tx.uat.cityba.se/login","credit-card" , "Manage Wallet");
```

Comment:
Not a big deal but your button id's are the same and the comma spacing is off for the button label.

Reply:
I would re-commit this code without the Citybase API code.